contrib/styles.scss

```diff
@@ -0,0 +1,298 @@
+  text-align: center;
+}
+
+h3 {
+  margin: 1em;
+}
+
+pre {
+  line-height: 1.6em;
+}
+
+a,
+a:visited {
+  color: inherit;
+}
+
+.code {
+  @include outer-container();
+  background-color: $lightblue;
+  padding: 0.5em;
+  text-align: center;
+}
+
+.page-section {
+  @include outer-container;
+  background-color: $lightblue;
+  line-height: $base-line-height * 1.5;
+  margin-bottom: 3em;
+  text-align: center;
+}
+
+.page-section-inner {
+  background-color: $blue;
+  font-size: 0.8em;
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  margin-bottom: 2em;
+  padding: 1em;
+  text-align: left;
+}
+
+// Zero
+// ============================================
+
+.zero-alpha,
+.zero-beta {
+  @include span-columns(6);
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  padding: 1em;
+  text-align: left;
+}
+
+// First
+// ============================================
+
+.first-alpha {
+  @include span-columns(1);
+}
+
+.first-beta {
+  @include span-columns(11);
+}
+
+.first-gamma {
+  @include span-columns(2);
+}
+
+.first-delta {
+  @include span-columns(10);
+}
+
+.first-epsilon {
+  @include span-columns(3);
+}
+
+.first-zeta {
+  @include span-columns(9);
+}
+
+.first-eta {
+  @include span-columns(4);
+}
+
+.first-theta {
+  @include span-columns(8);
+}
+
+.first-iota {
+  @include span-columns(5);
+}
+
+.first-kappa {
+  @include span-columns(7);
+}
+
+.first-lambda {
+  @include span-columns(6);
+}
+
+.first-mu {
+  @include span-columns(6);
+}
+
+.first-beta,
+.first-delta,
+.first-zeta,
+.first-theta,
+.first-kappa,
+.first-mu {
+  @include omega;
+}
+
+// Second
+// ============================================
+
+.second-alpha {
+  @include span-columns(4);
+}
+
+.second-beta {
+  @include span-columns(8);
+  line-height: $base-line-height * 2;
+  padding: 0;
+  text-align: center;
+}
+
+.second-beta-alpha,
+.second-beta-beta {
+  @include span-columns(4 of 8);
+  background-color: $lightblue;
+  height: 84%;
+  padding: 1em;
+}
+
+// Third
+// ===================================
+
+.third-dummy {
+  border-bottom: 0;
+  height: 5.5em;
+  text-align: center;
+}
+
+.third-alpha {
+  @include fill-parent();
+  @include row(table);
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.third-alpha-alpha {
+  @include span-columns(4);
+  border-right: 1px solid rgba(255,255,255, 0.5);
+}
+
+.third-alpha-beta {
+  @include reset-display;
+  @include span-columns(8);
+  height: 20em;
+}
+
+// Fourth
+// ===================================
+
+.fourth {
+
+  .fourth-alpha {
+    @include span-columns(6);
+    @include shift(3);
+    height: 20em;
+    text-align: center;
+  }
+}
+
+// Fifth
+// ===================================
+
+.fifth-box {
+  @include omega(4n);
+  @include span-columns(3);
+  font-size: 0.78em;
+  height: 8em;
+  margin-top: 2em;
+}
+
+// Fifth
+// ===================================
+
+.sixth {
+
+  pre {
+    padding: 10px 0;
+  }
+
+  @include media($medium-screen) {
```

Comment:
Expected item on line 270 to appear before line 266. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets